### ISSUE TITLE
How to query user information by ID

### ISSUE THREAD

**Issue by coke3636** [keybase/client#22884]
Adopt http api find user innformation by id
 For example： https://keybase.io/_/api/1.0/user/autocomplete.json?q=31AFBBE0A52FA1ED
Error accessing this URL 404
Previously, you can get the user's information through the HTTP get request of the car. How to use autocomplete

**Comment by wyllys66** [comment]
Yes, please.  This used to work.  If it is deprecated, please document what API to use for a substring search that will return a list of possible hits.
